finalPrep/modZeroHero.js

```diff
@@ -55,30 +74,61 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
-
+var scaryMonster = {
+  name: 'Bingus',
+  smell: 'stinky',
+  weight: 5000,
+  citiesDestroyed: ['Denver','Thornton','Pueblo'],
+  luckyNumbers: [5,7,85],
+  address: {
+    street: '4322 Clems St',
+    state: 'Colorado',
+    city: 'bugs',
+    zipCode: 83441
+  }
+}
+console.log(scaryMonster)
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
-
+//   - energyLevel = 50
+class SuperHero {
+  constructor(name,superpower,age){
+  this.name = name;
+  this.power = superpower;
+  this.age = age;
+  this.archNemesis = 'The Syntax Error';
+  this.powerLevel = 100;
+  this.energyLevel = 50;
+}
+sayName(sn){
+  console.log(this.name)
+}
+maximizeEnergy(me){
+  this.energylevel * 10
```

Comment:
Don't forget to re-assign the value to the `this.energyLevel` variable, ie. `this.energyLevel = this.energyLevel * 10`